binr/r2r/r2r.c

```diff
@@ -466,12 +466,14 @@ int main(int argc, char **argv) {
 	free (json_test_file);
 	free (fuzz_dir);
 #if __WINDOWS__
-	(void)SetConsoleOutputCP (old_cp);
-	// chcp doesn't pick up the code page switch for some reason
-	char *chcp = r_str_newf ("chcp %u > NUL", old_cp);
-	if (chcp) {
-		system (chcp);
-		free (chcp);
+	if (old_cp) {
+		(void)SetConsoleOutputCP (old_cp);
+		// chcp doesn't pick up the code page switch for some reason
+		char *chcp = r_str_newf ("chcp %u > NUL", old_cp);
```

Comment:
`r_sys_cmdf` instead of this?

Reply:
ok 1f27394f002d8faa32846ff36cc7c85c70ecb5d5